rpc-server/src/utils.rs

```diff
@@ -596,13 +596,13 @@ pub enum ShadowDataConsistencyError {
 #[derive(Debug)]
 pub enum DataMismatchReason {
     /// ReadRPC returns success result and NEAR RPC returns success result but the results mismatch
-    ReadRpcSuccessNearRpcSuccess,
+    SuccessNearRpcSuccess,
```

Comment:
This fixes bad naming practice, but it can cause some issues with metrics if it relies on old name